src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -469,23 +435,23 @@ public void selectQuestionNumber(int qnNumber, int newQnNumber){
      * {@code False} if not.
      */
     public boolean verifyEditSessionBoxIsEnabled() {
-        boolean isEditSessionEnabled = fsSaveLink.isDisplayed() && timezoneDropDown.isEnabled();
-        isEditSessionEnabled &= neverSessionVisibleTimeButton.isEnabled();
-        isEditSessionEnabled &= defaultSessionVisibleTimeButton.isEnabled();
-        isEditSessionEnabled &= customSessionVisibleTimeButton.isEnabled();
+        boolean isEditSessionEnabled = fsSaveLink.isDisplayed() && timezoneDropDown.isEnabled()
+                                       && neverSessionVisibleTimeButton.isEnabled()
+                                       && defaultSessionVisibleTimeButton.isEnabled()
+                                       && customSessionVisibleTimeButton.isEnabled()
+                                       // ...&&...
```

Comment:
I think short-circuiting is fine. Make the comment more explanatory. Have one comment for each logical section?